docs.html

```diff
@@ -0,0 +1,552 @@
+                </code>
+            </pre>
+        </div>
+        <!--Alert-->
+        <h2>Alert</h2>
+        <p class="paragraph-text">Documentation and Examples on BuilderUI Alerts</p>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <div>
+                <p class="alert alert-error">
+                    There was an error processing your request
+                </p>
+                <p class="alert alert-success">
+                    Data uploaded to server. Fire on!
+                </p>
+                <p class="alert alert-warning">
+
+                    There's something wrong. Could you please check again.
+                </p>
+                <p class="alert alert-info">
+
+                    There's something wrong. Could you please check again.
+                </p>
+            </div>
+        </div>
+        <h4>Code Examples</h4>
+        <div class="code-snippet">
+            <pre><code class="language-html">
+        &ltp class="alert alert-error"&gt
+            There was an error processing your request
+        &lt/p&gt
+        &ltp class="alert alert-success"&gt
+            Data uploaded to server. Fire on!
+        &lt/p&gt
+        &ltp class="alert alert-warning"&gt
+            There's something wrong. Could you please check again.
+        &lt/p&gt
+        &ltp class="alert alert-info"&gt
+            There's something wrong. Could you please check again.
+        &lt/p&gt
+            </p></p></code></pre>
+        </div>
+
+        <!--Avatar-->
+
+        <h2 id="avatar">Avatar</h2>
+        <p class="paragraph-text">Documentation and Examples on BuilderUI Alerts</p>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <img class="avatar" src="https://via.placeholder.com/50">
+            <img class="avatar-small" src="https://via.placeholder.com/50">
+        </div>
+        <h4>Code Snippet</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+        &ltimg class="avatar" src="https://via.placeholder.com/50&gt
+        &ltimg class="avatar-small" src="https://via.placeholder.com/50"&gt
+                </code>
+            </pre>
+        </div>
+
+        <!--Badge-->
+        <h2 id="badge">Badge</h2>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <span class="badge badge-default">default</span>
+            <span class="badge badge-success">success</span>
+            <span class="badge badge-error">error</span>
+            <span class="badge badge-new">default</span>
+            <span class="badge badge-notification">5</span>
+
+            <h4>
+                Badge on Icons/Avatars
+            </h4>
+            <button class="icon-button">
+                <span class="material-icons">
+                    notifications
+                </span>
+                <span class="badge-notification2">25</span>
+            </button>
```

Comment:
Different variations of the same components can be moved to their separate example boxes instead of nesting all in one.